apps/emqx/src/emqx_broker_sup.erl

```diff
@@ -59,4 +66,4 @@ init([]) ->
         modules => [emqx_broker_helper]
     },
 
-    {ok, {{one_for_all, 0, 1}, [BrokerPool, SharedSub, Helper]}}.
+    {ok, {{one_for_all, 0, 1}, [BrokerPool, SyncerPool, SharedSub, Helper]}}.
```

Comment:
As `BrokerPool` depends on `SyncerPool`, would it make sense to start `SyncerPool` before 'BrokerPool` ?  
It should not create any problems actually, since listeners are started only after all theses pools, so this is more like extra safety suggestion..

Reply:
Oh right, indeed.